docker-compose.yml

```diff
@@ -53,6 +53,8 @@ services:
       - RELAYER_TARGET_CHAIN_OUTPUT_FORMAT=json
       - RELAYER_TARGET_CHAIN_SIGN_MODE_STR=direct
       - RELAYER_REGISTRY_ADDRESSES=
+      - RELAYER_NEUTRON_CHAIN_ALLOW_KV_CALLBACKS=true
+      - RELAYER_ALLOW_TX_QUERIES=true
```

Comment:
out of scope, but I don't like the way we've designed these variables. why one of them is inside the neutron chain's config, and the other one is inside the relayer's config? I think they both should be on the relayer's level bc they're not related to the neutron chain, they're about the relayer configuration. do you agree? if yes, would it hurt if we fix it here?

Reply:
I think it's really out of scope of this PR. So let's have separate task. Would you mind creating this one

Reply:
sure